pkg/kubelet/apis/kubeletconfig/types.go

```diff
@@ -44,9 +44,9 @@ const (
 type KubeletConfiguration struct {
 	metav1.TypeMeta
 
-	// podManifestPath is the path to the directory containing pod manifests to
-	// run, or the path to a single manifest file
-	PodManifestPath string
+	// podPath is the path to the directory containing pods to
```

Comment:
nit: add `local (static)` here as well?

Reply:
is this meant to match v1beta1?

Reply:
We shouldn't lock in to the name "Static" either.

Reply:
> is this meant to match v1beta1?

yes